app/src/main/res/values/strings.xml

```diff
@@ -588,4 +590,9 @@
     <string name="health_authority">Health Authority:</string>
     <string name="facility">Facility:</string>
 
+    <!--ACCESSIBILITY STRINGS-->
+    <string name="banner_icon">banner icon</string>
+    <string name="down_arrow">down arrow</string>
+    <string name="right_arrow">right arrow</string>
+    <string name="login_card_image">login card image</string>
```

Comment:
This description doesn't provide relevant information to the user. I suggest removing it

Reply:
done